README.md

```diff
@@ -43,6 +43,7 @@ I *highly* recommend against using that globals build as it's quite strange you'
 * [@debounce](#debounce)
 * [@throttle](#throttle)
 * [@time](#time)
+* [@once](#once)
```

Comment:
ok, right.. 
should I remove older "new" labels? what's the `readme` routine here?